microsoftgraph/microsoft-graph-explorer-v4
# Fix margin-top on resources tab

## Overview
Fixes #1824 

### Demo
Margin is now consistent between samples, history and resources tab
<img width="226" alt="image" src="https://user-images.githubusercontent.com/45680252/173308330-7613a5f4-69c3-4eb5-a2e5-c4bcb42d4d89.png">
<img width="231" alt="image" src="https://user-images.githubusercontent.com/45680252/173308368-ddcdd7dc-5c00-45b5-8beb-3155985f716b.png">
<img width="223" alt="image" src="https://user-images.githubusercontent.com/45680252/173308401-a0437d8c-3788-48c8-8116-587a7dfdf7ba.png">


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Check out this branch and run it locally
Confirm the position of the search-box on the samples tab
Click on the resources tab. Notice that the position of the search-box is consistent with the samples tab and history tab
